yaki_backend/src/__test__/__mocks__/team.ts

```diff
@@ -1,16 +1,20 @@
 import {TeamDtoIn} from "../../features/team/team.dtoIn";
 
-const mockTeam: TeamDtoIn[] = [
+const teamMock: TeamDtoIn[] = [
   {
     teamId: 1,
+    teamName: "Red team",
     teamCaptainId: 1,
-    teamName: "Red Team",
+    teamCustomerId: 1,
+    teamActifFlag: true,
   },
   {
     teamId: 2,
+    teamName: "Blue team",
     teamCaptainId: 2,
```

Comment:
not suppose to become a list ? a team take a list of captain (or the conversion is done after ?)

Reply:
deleted file, not used anymore